src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -265,7 +287,13 @@ class BaseOptionsSelector extends Component {
                 showTitleTooltip={this.props.showTitleTooltip}
                 isDisabled={this.props.isDisabled}
                 shouldHaveOptionSeparator={this.props.shouldHaveOptionSeparator}
-                onLayout={this.props.onLayout}
+                onLayout={() => {
+                    this.scrollToIndex(this.state.focusedIndex, false);
+
+                    if (this.props.onLayout) {
+                        this.props.onLayout();
```

Comment:
@rushatgabhane pushed the fix. It's working fine as below:

https://user-images.githubusercontent.com/113963320/215655398-cb85b5c2-23b0-4758-9c55-ff3824ad0690.mp4

The reason was because of the `scrollToIndex` is running inside `onLayout`, for Android the list will appear for a moment before `onLayout` is called, thus causing the flicker.

The update was to run the `scrollToIndex` on both `componentDidMount` and `onLayout`. We still need the `scrollToIndex` call in `onLayout` because on iOS, the `scrollToLocation` API does not work if the layout was not calculated yet.

Reply:
@rushatgabhane since this is close to passing the 6 days milestone since the issue was assigned, I'd appreciate that we could coordinate to move this quickly.

Thanks!

Reply:
@rushatgabhane since it's close to passing the 6-days milestone since the issue was assigned, I'd appreciate if we can coordinate to move this quickly.

Thanks!